### PR TITLE
[Core] Extend Integration points extrapolation process for mpi

### PR DESCRIPTION
**Pull request by sunethwarna**
I was using this process in mpi, and realize that some of the methods for mpi is missing (so my answers in omp and mpi is way off). So I added mpi methods as I see is required to perform the same calculation in mpi. (but still mpi result doesn't match omp result perfectly, but they are close. I guess this is because of communication tolerances ryt?)

### CODE REVIEW

File: kratos/processes/integration_values_extrapolation_to_nodes_process.cpp [hunk 1]
```diff
@@ -217,6 +217,47 @@ void IntegrationValuesExtrapolationToNodesProcess::ExecuteFinalizeSolutionStep()
...
+    {
+        for (const auto p_var : mDoubleVariable)
+        {
+            mrModelPart.GetCommunicator().AssembleCurrentData(*p_var);
+        }
+        for (const auto p_var : mArrayVariable)
+        {
+            mrModelPart.GetCommunicator().AssembleCurrentData(*p_var);
+        }
+        for (const auto p_var : mVectorVariable)
```
Comment (philbucher) [inline kratos/processes/integration_values_extrapolation_to_nodes_process.cpp]:
```suggestion
        for (const auto p_var : mVectorVariable) {
```

should be on same line

**Review by philbucher — DISMISSED** [review]
Guess you are using it in vtk?
I was wondering who might be the first to try this :)

How much off are the results?

**Comment by sunethwarna** [comment]
I found the bug and fixed it.. Now it perfectly matches with OMP result :)

**Review by philbucher — DISMISSED** [review]
cool :)

**Review by philbucher — APPROVED** [review]

**Pull request merged**